ext/curl/curl.stub.php

```diff
@@ -74,9 +74,8 @@
 const CURLOPT_DNS_CACHE_TIMEOUT = UNKNOWN;
 /**
  * @var int
- * @cvalue CURLOPT_DNS_USE_GLOBAL_CACHE
  */
-const CURLOPT_DNS_USE_GLOBAL_CACHE = UNKNOWN;
+const CURLOPT_DNS_USE_GLOBAL_CACHE = -1;
```

Comment:
Is this to suppress a deprecation warning when building ext/curl against recent libcurl versions, or just a convention?

Reply:
Ah. just seen https://github.com/php/php-src/pull/5094#discussion_r368310001, so this is probably a good idea.